File: Phyton/Vetor/alturas.py
n = int(input("Quantas pessoas serão digitadas? "))
nome = [0 for x in range(n)]
idade = [0 for x in range(n)]
altura = [0 for x in range(n)]
media = 0
menor = 0
for i in range(n):
    print(f"Dados da {i+1}ª pessoa: ")
    nome[i] = input("Nome: ")
    idade[i] = int(input("Idade: "))
    altura[i] = float(input("Altura: "))
    media += altura[i]
    if idade[i] < 16:
        menor += 1

print(f"Altura média: {media/n:.2f}")
print(f"Pessoas com menos de 16 anos: {(menor*100)/n:.1f}%")

for i in range(n):
    if idade[i] < 16:
        print(nome[i])

File: C/Vetor/Alturas.c
#include <stdio.h>

void limpar_entrada() {
 char c;
 while ((c = getchar()) != '\n' && c != EOF) {}
}

int main(){
    int i, n;

    printf("Quantas pessoas serao digitadas? ");
    scanf("%d", &n);

        int idade[n];
        char nome[n][50];
        double altura[n], Amedia=0, Iporcentagem=0;

            for(i=0; i<n; i++){
                printf("Dados da %da pessoa:\n", i+1);
                printf("Nome :");
                limpar_entrada();
                gets(nome[i]);
                printf("Idade :");
                scanf("%d", &idade[i]);
                printf("Altura :");
                scanf("%lf", &altura[i]);
                }
            for(i=0; i<n; i++){
                Amedia = Amedia + altura[i];
                    if(idade[i] < 16){
                        Iporcentagem = Iporcentagem + 1;
                    }
            }
                Amedia = Amedia/n;
                Iporcentagem = (Iporcentagem * 100)/n;

                    printf("\nAltura media: %.2lf\n", Amedia);
                    printf("Pessoas com menos de 16 anos: %.1lf %%\n", Iporcentagem);

                        for(i=0; i<n; i++){
                            if (idade[i] < 16){
                                printf("%s\n", nome[i]);
                            }
                        }
return 0;
}
